packages/bpk-component-swap-button/src/BpkSwapButton.d.ts

```diff
@@ -16,14 +16,20 @@
  * limitations under the License.
  */
 
+export declare const SWAPBUTTON_STYLES: {
+    surfaceContrast: string;
+    canvasDefault: string;
+    canvasContrast: string;
+};
+export type SwapButtonStyle = (typeof SWAPBUTTON_STYLES)[keyof typeof SWAPBUTTON_STYLES];
 export type Props = {
-  ariaLabel: string;
-  ariaLiveTextProps: string;
```

Comment:
this type disappears, is intentional?

Reply:
Yes this was removed in a previous PR from Luna as it is handled on the consumer side but I missed in my review: https://github.com/Skyscanner/backpack/pull/3197